.github/workflows/release.yml

```diff
@@ -35,9 +54,18 @@ jobs:
       - name: Build
         run: NX_CLOUD_NO_TIMEOUTS=true pnpm build
       - name: Release
-        run: pnpm run-many --target=release:mainline --all --parallel=4 --output-style stream --nx-bail
+        run: pnpm exec release
       - name: Check for mutations
         run: git diff --ignore-space-at-eol --exit-code
+      - name: Tag last-release
+        shell: bash
+        run: |
+          git tag -f last-release
+          git push origin last-release --force
+      - name: Print release tags
+        run: git tag --list --contains `git rev-parse HEAD`
+      - name: Push release tags
+        run: git push origin --tags
```

Comment:
Need to remove this - we do not want to push all local tags since could include "minVersion" markers.